pages/strategy.md

```diff
@@ -0,0 +1,58 @@
+Exercism is a large complex project with thousands of contributors working across dozens of projects, all trying to achieve ambitious goals. It would quickly become overwhelming if we were all working towards making Exercism perfect, so to keep things manageable, we periodically determine the critical next step for Exercism to achieve its goals and then plan that into a strategy. Our current goal is to make Exercism scalable. We expect our next three goals to be to make Exercism financially sustainable, to make Exercism more accessible to beginners, and to intentionally grow Exercism for the first time.
```

Comment:
As this is a long read, I would personally prefer that the goals are separate from this big paragraph:

```suggestion
Exercism is a large complex project with thousands of contributors working across dozens of projects, all trying to achieve ambitious goals. It would quickly become overwhelming if we were all working towards making Exercism perfect, so to keep things manageable, we periodically determine the critical next step for Exercism to achieve its goals and then plan that into a strategy.

Our _current_ goal is to make Exercism scalable.

We expect our next three goals to be:
- to make Exercism financially sustainable,
- to make Exercism more accessible to beginners,
- and to intentionally grow Exercism for the first time.
```
------------

Exercism is a large complex project with thousands of contributors working across dozens of projects, all trying to achieve ambitious goals. It would quickly become overwhelming if we were all working towards making Exercism perfect, so to keep things manageable, we periodically determine the critical next step for Exercism to achieve its goals and then plan that into a strategy.

Our _current_ goal is to make Exercism scalable.

We expect our next three goals to be:
- to make Exercism financially sustainable,
- to make Exercism more accessible to beginners,
- and to intentionally grow Exercism for the first time.